tests/python/rest_api/test_tasks.py

```diff
@@ -817,5 +824,28 @@ def test_cannot_create_task_with_mythical_cloud_storage_data(
             "server_files": cloud_storage_content,
         }
 
-        status = self._test_cannot_create_task(self._USERNAME, task_spec, data_spec, org=org)
-        assert mythical_file in status.message
+        task_id = TestPostTaskData._test_create_task(
+            self._USERNAME, task_spec, data_spec, content_type="application/json", org=org
+        )
+
+        # save image from the "public" bucket and remove it temporary
+
+        s3_client = s3.make_client()
+        bucket_name = cloud_storages[cloud_storage_id]["resource"]
+
+        image = s3_client.download_fileobj(bucket_name, image_name)
+        s3_client.remove_file(bucket_name, image_name)
+
+        with make_api_client(self._USERNAME) as api_client:
+            try:
+                api_client.tasks_api.retrieve_data(
+                    task_id, number=0, quality="original", type="frame"
+                )
+                raise AssertionError("Frame should not exist")
+            except AssertionError:
+                raise
+            except Exception as ex:
+                assert ex.status == HTTPStatus.NOT_FOUND
+                assert image_name in ex.body
+
+        s3_client.create_file(bucket_name, image_name, image)
```

Comment:
Will the file be restored if the test fails?  Can this file affect other tests in this case?

Reply:
Fixed